### ISSUE TITLE
[Optim] Reduce amount of TransData Operators

### ISSUE THREAD

**Issue by zhangtao0408** [zhangtao0408/SeedVR#7]
# Problem Description

<img width="1101" height="60" alt="Image" src="https://github.com/user-attachments/assets/69a9141e-03ad-4dc6-8eb8-d22f00fbd48a" />

The current model contains a significant number of TransData during inference after each conv ops. These operators account for 7.27%​ of the total network runtime, representing a potential performance bottleneck. On certain hardware accelerators (e.g., NPU/GPU), such operations can introduce unnecessary overhead. Removing or reducing these operators is expected to improve inference efficiency.

# Current Status

<img width="1814" height="242" alt="Image" src="https://github.com/user-attachments/assets/cb77b6f1-a3ba-48e9-85cb-534e6d9a86f0" />

- Profiling results show TransData operator time consumption: 7.27%​ of total runtime.

# Optimization Goals

- Reduce or eliminate TransData operators in the model.
- Maintain functional equivalence and numerical accuracy

# Possible Solution

We can enable internal format computation in the PyTorch NPU backend to minimize TransDataoperations. This allows the framework to optimize data layout internally without explicit conversion operators:

```python
import torch
import torch_npu

# Enable JIT compilation for optimization
torch.npu.set_compile_mode(jit_compile=True)

# Allow internal format computation to reduce explicit TransData operations
torch_npu.npu.config.allow_internal_format = True
```
